prody/tests/proteins/test_header.py

```diff
@@ -39,8 +39,112 @@ def tearDown(self):
 
         self.header = None
 
+class TestParsePDBHeaderCRYST1andSCALE(unittest.TestCase):
+
+    def setUp(self):
+        self.header = parsePDB(pathDatafile('1pwc.pdb'),
+                               header=True, model=0)
+
+    def testHeaderType(self):
```

Comment:
this feels like something that should go in the general header test

Reply:
Here I am not sure what you mean. I putt his in its own test as it operates on a different pdb file than TestParsePDBHeaderOnly. Do you want em to create header1 and header2 in the setup and then perform the tests on the 2 header in the test function ?

Reply:
I mean we probably don’t need to check the type of the header for every pdb file, but it’s probably fine to keep too.

you can keep your header1 and header2 in separate tests, I guess